### PR TITLE
Profile settings

### PR DESCRIPTION
**Pull request by remkst777**

### CODE REVIEW

File: app/containers/ChangeEmail/ChangeEmailForm.jsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      <FormattedMessage {...signUpMessages.email} />
+    </H4>
+
+    <form onSubmit={handleSubmit(changeEmail)}>
+      <Field
+        name={NEW_EMAIL_FIELD}
+        disabled={changeEmailProcessing}
+        label={translationMessages[locale][changeEmailMessages.newEmail.id]}
+        component={TextInputField}
+        validate={[validateEmail, strLength3x20, required]}
```
Comment (sergeyi23) [inline app/containers/ChangeEmail/ChangeEmailForm.jsx]: `strLength3x20` should not be needed here. `validateEmail` has to check the length as well for simplicity.

File: app/containers/ChangeEmail/ChangeEmailForm.jsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      />
+
+      <Field
+        name={CONFIRM_EMAIL_FIELD}
+        disabled={changeEmailProcessing}
+        label={
+          translationMessages[locale][changeEmailMessages.confirmNewEmail.id]
+        }
+        component={TextInputField}
+        validate={[validateEmail, strLength3x20, required]}
```
Comment (sergeyi23) [inline app/containers/ChangeEmail/ChangeEmailForm.jsx]: Same as above

File: app/containers/ChangeEmail/saga.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+export function* sendOldEmailWorker({ resetForm, email }) {
+  try {
+    const locale = yield select(makeSelectLocale());
+    const translations = translationMessages[locale];
+
+    const response = yield call(() => changeCredentialsInit(email));
+
+    if (!response.OK) {
+      throw new Error(
+        translations[webIntegrationErrors[response.errorCode].id],
```
Comment (sergeyi23) [inline app/containers/ChangeEmail/saga.js]: What will happen here is webIntegrationErrors does not contain a value for error code?
do you think it makes sense to create a method that returns a translation for error code; and returns some default message if there is nothing for that error code?

File: app/containers/ChangeEmail/ChangeEmailForm.jsx (hunk 1 above)
Comment (remkst777) [reply app/containers/ChangeEmail/ChangeEmailForm.jsx]: fixed

File: app/containers/ChangeEmail/ChangeEmailForm.jsx (hunk 2 above)
Comment (remkst777) [reply app/containers/ChangeEmail/ChangeEmailForm.jsx]: fixed

**Pull request merged**